gandalf.js

```diff
@@ -1,128 +1,125 @@
 /**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?